operations/commit_queue.go

```diff
@@ -215,29 +217,57 @@ func listCommitQueue(ctx context.Context, client client.Communicator, ac *legacy
 }
 
 func listPRCommitQueueItem(ctx context.Context, item restModel.APICommitQueueItem, projectRef *model.ProjectRef, uiServerHost string) {
+	issue := restModel.FromAPIString(item.Issue)
+	displayPRAuthor(ctx, projectRef, issue)
```

Comment:
Shouldn't this also be called from `listCLICommitQueueItem`?

Reply:
Well this is just to break up the logic for printing the author for Pull Requests (hence the function name). For CLIs it's much easier: p.Author :)

Reply:
Oh, totally missed that, sorry.